metaflow/flowspec.py

```diff
@@ -354,7 +358,12 @@ def _find_input(self, stack_index=None):
                     )
             return self._cached_input[stack_index]
 
-    def merge_artifacts(self, inputs, exclude=[], include=[]):
+    def merge_artifacts(
+        self,
+        inputs: Sequence[TFlowSpec],
```

Comment:
Similar question to elsewhere: why `TFlowSpec` instead of `FlowSpec`? Feel free to ignore this question if answered elsewhere.